translations/README.spanish.md

```diff
@@ -0,0 +1,216 @@
+| Azure | [![Implementar en Azure](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3a%2f%2fraw.githubusercontent.com%2fhasura%2fgraphql-engine%2fmaster%2finstall-manifests%2fazure-container-with-pg%2fazuredeploy.json) | [docs](https://docs.hasura.io/1.0/graphql/manual/guides/deployment/azure-container-instances-postgres.html) |
+
+### Otros métodos de implementación
+
+Para la implementación basada en Docker y las opciones de configuración avanzadas, consulte [implementación guías](https://docs.hasura.io/1.0/graphql/manual/getting-started/index.html) o
+[instalar manifiestos](../install-manifests).
+
+## Arquitectura
+
+El motor GraphQL de Hasura presenta una instancia de base de datos Postgres y puede aceptar solicitudes GraphQL de sus aplicaciones cliente. Se puede configurar para que funcione con su sistema de autenticación existente y puede manejar el control de acceso utilizando reglas de nivel de campo con variables dinámicas de su sistema de autenticación.
+
+También puede fusionar esquemas GraphQL remotos y proporcionar una API GraphQL unificada.
+
+![Arquitectura del motor Hasura GraphQL](../assets/hasura-arch.svg)
+
+## Herramientas del lado del cliente
+
+Hasura funciona con cualquier cliente GraphQL. Recomendamos utilizar [Apollo Client](https://github.com/apollographql/apollo-client). Consulte [awesome-graphql](https://github.com/chentsulin/awesome-graphql) para obtener una lista de clientes.
+
+## Añadir lógica de negocios
+
+GraphQL Engine proporciona métodos fáciles de razonar, escalables y de rendimiento para agregar lógica empresarial personalizada a su backend:
+
+### Esquemas remotos
+
+Agregue resolvers personalizados en un esquema remoto además del esquema GraphQL basado en Postgres de Hasura. Ideal para casos de uso como implementar una API de pago o consultar datos que no están en su base de datos - [leer más](../remote-schemas.md).
+
+### Activar webhooks en eventos de base de datos
+
+Agregue lógica de negocios asíncrona que se desencadena en función de los eventos de la base de datos.
+Ideal para notificaciones, canalizaciones de datos de Postgres o asíncronas.
+procesamiento - [leer más](../event-triggers.md).
+
+### Datos derivados o transformaciones de datos
+
+Transforme los datos en Postgres o ejecute la lógica de negocios en ellos para derivar otro conjunto de datos que se pueda consultar con GraphQL Engine - [leer más](https://docs.hasura.io/1.0/graphql/manual/queries/derived-data.html).
+
+## Demos
+
+Consulte todas las aplicaciones de ejemplo en
+Directorio [community/examples](../community/examples).
```

Comment:
This link is broken. Could you add the correct one please?

Reply:
Mam in latest master these does not exist 
I think examples folder has been removed